Source/WebCore/rendering/RenderListMarker.h

```diff
@@ -39,8 +39,8 @@ class RenderListMarker final : public RenderBox {
     RenderListMarker(RenderListItem&, RenderStyle&&);
     virtual ~RenderListMarker();
 
-    StringView textWithoutSuffix() const;
-    StringView textWithSuffix() const { return m_textWithSuffix; }
+    StringView textWithoutSuffix() const { return m_textContent.textWithoutSuffix; };
+    StringView textWithSuffix() const { return m_textContent.textWithSuffix(); };
```

Comment:
This is unsafe. Can't safely return a view of a temporary `String`.

Reply:
Can you check if my newest version is correct? I've never had to deal with this problem so I'm not sure what the right pattern is here.

Reply:
Switched to just returning the String directly.